scripts/jenkins/ardana/gerrit/gerrit_merge.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import sys
+
+sys.path.append(os.path.dirname(__file__))
+from gerrit import GerritChange  # noqa: E402
+from gerrit_project_map import gerrit_project_map  # noqa: E402
+
+
+def check_dependencies(change):
+    change.load_dependencies()
+
+    change_deps = change.dependencies()
+
+    unmerged_deps = []
+    for change_dep in change_deps:
+        if change_dep.status != "MERGED":
+            unmerged_deps.append(change_dep)
+        else:
+            change_deps.extend(change_dep.dependencies())
+
+    if unmerged_deps:
+        print "Unmerged dependencies:\n{}".format('\n'.join([
+            str(c) for c in unmerged_deps]))
+        return False
+
+    return True
+
+
+def gerrit_merge(change_no):
```

Comment:
This should probably take in a change object too (see comments on gerrit_review.py)

Reply:
done. btw, this is great for reducing the number of Gerrit API calls we perform for a single set of operations.